src/components/admin/Department.vue

```diff
@@ -227,7 +228,8 @@ export default defineComponent({
     margin: 10px;
   }
   .department-container{
-    max-width: 1140px;
+    // max-width: 1140px;
+    max-width: 1163px;
```

Comment:
Why hard coded? 
It won't be responsive.

Reply:
It was already hard coded I just increase the max-width value

Reply:
I know. Please make it dynamic/responsive.